CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.1](https://github.com/brave/brave-ios/releases/tag/v1.57.1)
+
+- Resolved issue where Brave Ads are disabled if Brave News was already enabled. ([#8098](https://github.com/brave/brave-ios/issues/8098)) 
```

Comment:
Looks like there's an extra space at the end of the line. It will look fine if merged this way but lets remove it so it looks cleaner in the CHANGELOG file.

Reply:
https://github.com/brave/brave-ios/commit/3706c66bb8915375212a86e1939d74d2c8378468